src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started work");

```

Comment:
The message 'Truck started work' could be more informative. It should indicate what type of Machine is working. Consider including the specific action the Truck is performing.